stripecardscan-example/src/main/java/com/stripe/android/stripecardscan/example/MainActivity.kt

```diff
@@ -13,12 +21,50 @@ class MainActivity : AppCompatActivity() {
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        val requestQueue = Volley.newRequestQueue(this)
+
         super.onCreate(savedInstanceState)
         setContentView(viewBinding.root)
         setSupportActionBar(findViewById(R.id.toolbar))
 
+        val civLauncher = registerForActivityResult(
+            ActivityResultContracts.StartActivityForResult()
+        ) { result ->
+            viewBinding.scanResultText.text = result.data?.getStringExtra(PARAM_VERIFICATION_RESULT)
+        }
+
+        viewBinding.getPublishableKeyButton.setOnClickListener {
+            requestQueue.add(
+                JsonObjectRequest(
+                    "https://stripe-card-scan-civ-example-app.glitch.me/publishable_key",
+                    { response ->
+                        viewBinding.stripePublishableKey.setText(
+                            response.get("publishable_key") as String
+                        )
+                        viewBinding.launchCompleteButton.isEnabled = true
+                    },
+                    { error ->
+                        Toast.makeText(
```

Comment:
Might be better to use a snackbar rather than a toast. We had some issues seeing toasts pop up on emulators during our bug bash and thought we had an issue. 

https://github.com/stripe/stripe-android/blob/master/paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/viewmodel/PaymentSheetViewModel.kt#L26